web_listview_invert_selection/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2017 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'List Invert Selection',
+    'summary': """
+        Allows to invert the current selection of a list of records
+    """,
+    'version': '10.0.1.0.0',
+    'category': 'Web',
+    'author': 'Onestein,Odoo Community Association (OCA)',
+    'website': 'http://www.onestein.eu',
+    'license': 'AGPL-3',
+    'depends': [
+        'web',
+    ],
+    'data': [
+        'templates/assets.xml'
+    ],
+    'qweb': ['static/src/xml/web_listview_invert_selection.xml'],
+    'installable': True,
+    'application': False,
```

Comment:
You can remove this line (`'application': False,`): it will take `False` as default value